packages/block-editor/src/hooks/block-hooks.js

```diff
@@ -19,18 +19,28 @@ import { store as blockEditorStore } from '../store';
 
 const EMPTY_OBJECT = {};
 
-function BlockHooksControlPure( { name, clientId } ) {
+function BlockHooksControlPure( {
+	name,
+	clientId,
+	metadata: { ignoredHookedBlocks = [] } = {},
```

Comment:
That's something that might work pretty well for modified templates and parts 👍🏻

Reply:
Yeah, I think it should be pretty solid. Unfortunately, it won't work with unmodified ones.

Maybe we'll eventually reconsider and re-introduce setting the `ignoredHookedBlocks` metadata on read after all (which would make it work for unmodified ones); since we would also still need to keep it on write, we'd have to basically consider if it's worth the tradeoff of duplicating that logic.